L30-cpp-server-load-reporting.md

```diff
@@ -0,0 +1,42 @@
+﻿C++ API changes for Server Load Reporting
+----
+* Author(s): AspirinSJL
+* Approver: markdroth, vjpai
+* Status: Draft
+* Implemented in: C++
+* Last updated: 2018-07-13
+* Discussion at: TBA
+
+## Abstract
+
+As part of the load balancing functionality, the server-side load reporting service is to be introduced. A `LoadReportingServiceServerBuilderOption` will be used to enable that service. A helper function `AddLoadReportingCost()` can be used to report the customized call metrics.
+
+## Background
+
+A load balancer needs to get the load data of the servers in order to balance the load among the servers accordingly. One possible approach to get the load data is to request the load data from the servers via RPCs. That requires the servers to register an additional service to report the load data.
+
+
+### Related Proposals: 
+N/A
+
+## Proposal
+
+We propose a new public header `include/grpcpp/ext/server_load_reporting.h` to use the server load reporting service. Note that this public header (and the load reporting service) is only available when the binary is built with the `grpcpp_server_load_reporting` library. That library is only available in Bazel build because it depends on OpenCensus which can only be built with Bazel.
+
+1. The header contains a `LoadReportingServiceServerBuilderOption`. The user should set that option in the `ServerBuilder` to enable the load reporting service on a server.
+2. The header also contains a helper function `void AddLoadReportingCost(grpc::ServerContext* ctx, const grpc::string& cost_name, double cost_value);`. Besides the canonical call metrics (e.g., the number of calls started, the total bytes received), the user can use this function to add other customized call metrics from their own normal services.
```

Comment:
Why are you calling it a "helper function"? It's just a public free function, correct?

Reply:
Yes, but it's not essential. The users can implement one without much effort. It only does some serialization and calls the `ServerContext::AddTrailingMetadata()` to add the entries. 

I rephrased it to "utility function" which might be clearer.